src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -0,0 +1,389 @@
+.signpost {
+    display: inline;
+
+    &:hover {
+        cursor: pointer;
+    }
+
+    button {
+        min-width: $clr_baselineRem_1;
+        margin: 0 $clr_baselineRem_0_3;
+        padding: 0;
+        color: $clr-dark-midtone-gray!important;
+        outline: none;
+
+        clr-icon {
+            width: $clr_baselineRem_1;
+            height: $clr_baselineRem_1;
+        }
+
+        &:hover, &.active {
+            color: $clr-action-blue!important;
+        }
+    }
+
+    /***
+        The signpost is above the icon top-{HORIZONTAL_POSITION}
+    */
+    &.top-left {
+        .signpost-content {
+            .popover-pointer {
+                width: 0;
+                height: 0;
+                border-left: 12px solid transparent;
+                border-top: 12px solid #9a9a9a;
```

Comment:
We have a color variable in `variables.clarity.scss` that we can reuse here instead of `#9a9a9a` and same for the ones below. I think you didn't use the baseline rem variables for 12px here because this is for the pointer arrows right?